src/WebsockettestHelpers.js

```diff
@@ -105,3 +105,21 @@ Math.median = function(values) {
 };
 
 
+let Logger = (function () {
```

Comment:
That was intended as a simple example logger. You would provide it either with a configurable logger or one where the other levels are just empty functions.
The better approach would be to use a factory method, like java's `Logger.getLogger()`.